resources/views/components/cd/layout/app.blade.php

```diff
@@ -7,24 +7,24 @@
 
     <title>{{ $title }}</title>
 
-    <link rel="icon" href="{{ asset('cwd-framework/favicon.ico') }}" type="image/vnd.microsoft.icon"/>
+    <link rel="icon" href="{{ asset('cd/favicon.ico') }}" type="image/vnd.microsoft.icon"/>
```

Comment:
I wonder if we should split all of these items that are published as assets (that is, in /public) and still deliver them as cwd-framework. I feel like that would be more accurate and clear about what is coming from the outside theme. 

To be clear: we would keep the view components in our short-named directory (/resources/views/components/cd)

Reply:
I think those assets in /public should remain under cwd-framework